operating_versioned_docs/version-v0.5/manuals/collaset.md

```diff
@@ -0,0 +1,968 @@
+
+If update pod template with `InPlaceIfPossible` policy as following example, the Pod will not be recreated.
+
+``` shell
+$ kubectl -n default edit cls collaset-sample
+apiVersion: apps.kusionstack.io/v1alpha1
+kind: CollaSet
+metadata:
+  name: collaset-sample
+spec:
+  template:
+    ......
+    spec:
+      containers:
+      - image: nginx:1.24.0  # changed from nginx:1.25.2
+  ...
+  updateStrategy:
+    podUpgradePolicy: InPlaceIfPossible # use InPlaceIfPossible policy
+
+$ kubectl -n default get pod -o yaml | grep "image: nginx"
+    - image: nginx:1.24.0
+    - image: nginx:1.24.0
+    - image: nginx:1.24.0
+
+$ kubectl -n default get pod
+NAME                     READY   STATUS    RESTARTS     AGE
+collaset-sample-5wvlh    1/1     Running   1 (6s ago)   2m10s
+collaset-sample-ldvrg    1/1     Running   1 (6s ago)   2m10s
+collaset-sample-pbz75    1/1     Running   1 (6s ago)   2m10s
+```
+
+### Replace Update Pod
+
+CollaSet provides the `Replace` policy for certain applications that are sensitive to the available number of Pods.
+
+``` yaml
+apiVersion: apps.kusionstack.io/v1alpha1
+kind: CollaSet
+metadata:
+  name: collaset-sample
+spec:
+  ...
+  updateStrategy:
+    podUpgradePolicy: Replace  # Options: InPlaceIfPossible, Recreate, Replace
+```
+
+The `Replace` policy indicates that CollaSet should update a Pod by creating a new one to replace it. 
+Unlike the `Recreate` policy, which deletes the old Pod before creating a new updated one, or the `InPlaceIfPossible` policy, which updates the current Pod in place, 
+the `Replace` policy first creates a new Pod with the updated revision. It then deletes the old Pod once the new one becomes available for service.
+
+```shell
+# Before updating CollaSet
+$ kubectl -n default get pod
+NAME                    READY   STATUS        RESTARTS   AGE
+collaset-sample-dwkls   1/1     Running       0          6m55s
+
+# After updating CollaSet, the updated Pod is created first
+$ kubectl -n default get pod
+NAME                    READY   STATUS              RESTARTS   AGE
+collaset-sample-dwkls   1/1     Running             0          6m55s
+collaset-sample-rcmbv   0/1     ContainerCreating   0          0s
+
+# Once the created Pod is available for service, the old Pod will be deleted
+$ kubectl -n default get pod
+NAME                    READY   STATUS              RESTARTS   AGE
+collaset-sample-rcmbv   1/1     Running             0          1s
+collaset-sample-dwkls   1/1     Terminating         0          7m12s
+```
+
+The two Pods will have a pair of labels to identify their relationship. The new Pod will have the label `collaset.kusionstack.io/replace-pair-origin-name` to indicate the name of the old Pod, and the old Pod will have the label `collaset.kusionstack.io/replace-pair-new-id` to indicate the instance ID of the new Pod.
+
+Additionally, the new Pod and old Pod will each begin their own PodOpsLifecycles, which are independent of each other.
+
+### OperationDelaySeconds
```

Comment:
there was “Selective Pod Deletion” section in v0.4, no longer needed?

Reply:
mistake, recover “Selective Pod Deletion” section

Reply:
fixed by: https://github.com/KusionStack/kusionstack.io/pull/530/commits/8839062bc0e043f000e2b8f9123dc7ad26e91bf8